colossalai/kernel/cuda_native/csrc/attention_infer_kernels/rmsnorm/layernorm_kernels.cu

```diff
@@ -0,0 +1,62 @@
+/*This code from Vllm : https://github.com/vllm-project/vllm
```

Comment:
Should we add their license to ours?

Reply:
I am not quite sure. We can remove this temporarily

Reply:
NO. I mean we should add their license to ours (`ColossalAI/LICENSE`)